Userland/Libraries/LibJS/Tests/builtins/Object/Object.prototype.hasOwnProperty.js

```diff
@@ -10,4 +10,9 @@ test("basic functionality", () => {
     o.undefined = 2;
     expect(o.hasOwnProperty()).toBeTrue();
     expect(o.hasOwnProperty(undefined)).toBeTrue();
+
+    var test_symbol = Symbol("real");
+    o[test_symbol] = 3;
+    expect(o.hasOwnProperty(test_symbol)).toBeTrue();
+    expect(o.hasOwnProperty(Symbol("fake"))).toBeFalse();
```

Comment:
Please use pascalCase in JavaScript code, snake_case doesn't mix nicely with the built-in function names.

```suggestion
    var testSymbol = Symbol("real");
    o[testSymbol] = 3;
    expect(o.hasOwnProperty(testSymbol)).toBeTrue();
    expect(o.hasOwnProperty(Symbol("fake"))).toBeFalse();
```

Reply:
fixed! ill try to remember that for next time :)